system/Helpers/kint_helper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+// This helper is autoloaded by CodeIgniter.
+
+if (! function_exists('dd')) {
+    if (class_exists(Kint::class)) {
+        /**
+         * Prints a Kint debug report and exits.
+         *
+         * @param array ...$vars
+         *
+         * @codeCoverageIgnore Can't be tested ... exits
+         */
+        function dd(...$vars)
+        {
+            // @codeCoverageIgnoreStart
+            Kint::$aliases[] = 'dd';
+            Kint::dump(...$vars);
+
+            exit;
+            // @codeCoverageIgnoreEnd
+        }
+    } else {
+        // In case that Kint is not loaded.
+        function dd(...$vars)
+        {
+            return 0;
```

Comment:
This is potentially dangerous. If someone has been using `dd()` in production then they are likely relying on it terminating. I would propose a closer fallback:
```suggestion
            var_dump(...$vars);
            exit;
```

Reply:
Kint is disabled in production environment:
https://github.com/codeigniter4/CodeIgniter4/blob/2788a492ea9b2ae8c42873ed01ecf435df8d269c/system/CodeIgniter.php#L188-L190

So `dd()` returns `0`:
https://github.com/codeigniter4/CodeIgniter4/blob/2788a492ea9b2ae8c42873ed01ecf435df8d269c/system/ThirdParty/Kint/Kint.php#L558-L560

Reply:
Ah that makes sense! Thanks